server/src/main/java/org/elasticsearch/index/IndexingPressure.java

```diff
@@ -108,13 +114,17 @@ public class IndexingPressure {
     private final AtomicLong lowWaterMarkSplits = new AtomicLong(0);
     private final AtomicLong highWaterMarkSplits = new AtomicLong(0);
 
+    private final AtomicLong largeOpsRejections = new AtomicLong(0);
```

Comment:
My idea is to use this specific metrics related to large documents / expansion issues in autoscaling, that way we could have an idea of the extra memory that the nodes would need to cope with the load.

Reply:
++